tests/pytests/test_config.py

```diff
@@ -240,6 +240,70 @@ def testImmutable(env):
     env.expect('ft.config', 'set', 'RAW_DOCID_ENCODING').error().contains(not_modifiable)
     env.expect('ft.config', 'set', 'BG_INDEX_SLEEP_GAP').error().contains(not_modifiable)
 
+@skip(cluster=True, noWorkers=True)
+def testDeprecatedMTConfig():
+    env = Env(moduleArgs='WORKER_THREADS 3 MT_MODE MT_MODE_FULL')
```

Comment:
We should not create multiple envs on the same test, lets split it to multiple tests.